racingcar-java/src/test/java/mission02/CalculationTest.java

```diff
@@ -0,0 +1,78 @@
+package mission02;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CalculationTest {
+
+    Calculation calculation = new Calculation();
+    String[] types = {OperatorType.PLUS, OperatorType.MINUS, OperatorType.MULTIPLY, OperatorType.DIVIDED};
+
+    @DisplayName("연산기호 테스트 - +, -, *, /")
+    @ParameterizedTest
+    @ValueSource(strings = {"+", "-", "*", "/"})
+    void 연산기호가_올바른_사칙연산의_연산기호인지_확인한다(String type){ // 이거 리뷰 필요
+        // when
+        boolean check = Arrays.stream(types).anyMatch(type::equals);
+
+        // then
+        assertTrue(check);
+    }
+
+    @DisplayName("연산기호\'+\' 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {"1, 2, 3", "2, 3, 5", "5, 100, 105"})
+    void 두개의_값이_주어지고_합하면_올바른_값이_나온다(int baseOperand, int nextOperand, int result){
```

Comment:
```suggestion
    void 두개의_값이_주어지고_합하면_올바른_값이_나온다(int baseOperand, int nextOperand, int expected){
```
기댓값에 대한 네이밍이 어울릴 것 같네요